Core/MaterialParameters.cpp

```diff
@@ -60,6 +60,22 @@ const MaterialConstants MaterialParameters::fAir = MaterialConstants(
   {142 * ns}, {75}, 0.125 * ns, 1.0 / 3.0, {0.5 * ns}, {0}
 );
 
+const MaterialConstants MaterialParameters::fPolycarbonate = MaterialConstants( 
```

Comment:
Do you know what kind of polycarbonate it was, like type, purity anything?
Attachment: articles with positron lifetime in polycarbonate
https://pure.tue.nl/ws/portalfiles/portal/1711754/621436.pdf
https://aip.scitation.org/doi/10.1063/1.2957074
https://onlinelibrary.wiley.com/doi/pdf/10.1002/pssa.2211570218